ipywidgets/widgets/interaction.py

```diff
@@ -217,6 +217,11 @@ def call_f(name=None, old=None, new=None):
         if manual:
             manual_button.disabled = True
         try:
+            def close_result_widgets(w):
+                if type(w.result) is Box:
+                    close_result_widgets(w.result)
```

Comment:
That's what I initially expected, but when I tested it on the gist, `.close()` seemed to recursively close the `children` but not the `result` of a widget.